src/lib/Invitation/Persistence/DoctrineGateway.php

```diff
@@ -223,7 +227,7 @@ public function updateInvitation(string $hash, InvitationUpdateStruct $updateStr
         $query->where(
             $query->expr()->eq(
                 'hash',
-                $query->createNamedParameter($hash, ParameterType::STRING, ':hash')
+                $query->createNamedParameter($hash, ParameterType::STRING, ':current_hash')
```

Comment:
is `':current_hash'` correct?

Reply:
it is as this is named parameter and `hash` is used to provide new `hash` to update